### DISCUSSION TITLE
Set visibility/access to sanctum tokens feature based on callback

### DISCUSSION THREAD

**Discussion by JeremyDunn** [Jacobtims/filament-breezy#232] [Ideas]
I'd like to conditionally show/hide the sanctum token section on the my profile page based on the current user's permissions. 
Maybe the `enableSanctumTokens` method's condition parameter could accept a callback or boolean?

**Comment by dangnhdev** [comment]
I also want to have this feature, where only users with an **admin** or **dev** role can generate tokens.
